services/apigateway/src/test/java/software/amazon/awssdk/services/apigateway/internal/AcceptJsonInterceptorTest.java

```diff
@@ -0,0 +1,53 @@
+package software.amazon.awssdk.services.apigateway.internal;
```

Comment:
Thanks for creating the test class! Missing copyright header.

Reply:
Done!